jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/RetrieveRegistryCredentialsStep.java

```diff
@@ -196,10 +199,13 @@ Authorization retrieveFromCredentialHelper(String credentialHelperSuffix)
     buildLogger.info("Checking credentials from docker-credential-" + credentialHelperSuffix);
 
     try {
-      Authorization authorization =
+      Credentials credentials =
           dockerCredentialHelperFactory
-              .newDockerCredentialHelper(registry, credentialHelperSuffix)
+              .newDockerCredentialHelper(
```

Comment:
You still seems to have the suffix version of `newDockerCredentialHelper`. Why using the `Path` version?